scipy/optimize/_lsq/least_squares.py

```diff
@@ -231,8 +231,7 @@ def least_squares(
         loss_scale=1.0, scaling=1.0, diff_step=None, tr_solver=None,
         tr_options={}, jac_sparsity=None, max_nfev=None, verbose=0, args=(),
         kwargs={}):
-    """Minimize the robustified sum of squares of nonlinear functions, subject
-    to bound constraints on independent variables.
+    """Robust nonlinear least squares with bounds on the variables.
```

Comment:
Not having an action verb in the summary line is quite unusual. Is this something somebody else suggested?
If you don't like "minimize", maybe "solve" would be better?

Suggestion: "Solve a nonlinear least squares problem with bounds on the variables." Of course, there's no mention of "robust" as this does not fit, is this a big deal if this does not appear here?